packages/graph/hash_graph/lib/graph/src/ontology/mod.rs

```diff
@@ -59,100 +56,45 @@ impl PersistedOntologyIdentifier {
     }
 }
 
-#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize, Component)]
+fn serialize_ontology_type<T, S>(ontology_type: &T, serializer: S) -> Result<S::Ok, S::Error>
+where
+    T: Clone,
+    serde_json::Value: From<T>,
+    S: Serializer,
+{
+    // TODO: get rid of this clone
```

Comment:
If we want to use `serialize_with` then we need to take a reference here.

We could manually implement `Serialize` for the `Persisted*` types I guess but that doesn't seem ideal.

Reply:
I think in the long term we should do this, but it's definitely not worth doing it now.

Reply:
I have removed the TODO as I don't think we should put TODOs for optimizations that aren't backed by evidence. I have replaced it with a comment explaining the clone isn't really needed though.